apps/src/lib/kits/maker/portScanning.js

```diff
@@ -39,8 +40,9 @@ export function findPortWithViableDevice() {
       } else {
         return Promise.reject(
           new ConnectionFailedError(
-            'Did not find a usable device on a serial port. ' +
-              '\n\nFound devices: ' +
+            applabI18n.noUsableDeviceOnPort() +
+              '\n\n' +
+              applabI18n.foundDevices() +
```

Comment:
for string concatenation, i think best practice is to incorporate one of the strings into the other, if possible. i realize that makes no sense, but i can't think of a better way to say it, so i'll show you 😂 

for this example, we don't need to concatenate the first two strings because there are newlines that break the string up for us visually and that should work for all languages. but for the last two strings, we could make sure that they make sense in RTL by doing this:

```
// strings file
"foundDevices": "Found devices: {deviceList}"

// js
applabI18n.foundDevices({deviceList: JSON.stringify(list)})
```

there are a couple of places where i think this change will help, so i'll comment on the ones i notice